cmd/agent/app/integrations.go

```diff
@@ -342,10 +379,53 @@ func installTuf(cmd *cobra.Command, args []string) error {
 	)
 }
 
-func getIntegrationVersion(integration string, cachePath string) (string, error) {
+func getMinVersion(integration string) (versionStruct, error) {
+	here, _ := executable.Folder()
+	lines, err := ioutil.ReadFile(filepath.Join(here, relReqAgentReleasePath))
+	if err != nil {
+		return versionStruct{}, err
+	}
+	version, err := getVersionFromReqLine(integration, string(lines))
+	if err != nil {
+		return versionStruct{}, err
+	}
+
+	return version, nil
+}
+
+func isAbove(version1 versionStruct, version2 versionStruct) bool {
```

Comment:
I don't think `version1` and `version2` are good name. We are trying to compare previous and current version numbers, and should name them as much. Much less easier to make mistakes that way.

Reply:
I tried to make the function more like a generic operator, but if it's confusing I can switch back the naming to how it was.

Reply:
IMO caller usage like `version1.isAbove(version2)` would be clearer than `isAbove(version1, version2)`, so I'd make this function a method of the version struct

Reply:
On second thought, that should work. If programmers accidentally substitute previous with current version numbers, then that kind of logical error would be hard to prevent even with descriptive parameter names.